include/netutils/webclient.h

```diff
@@ -235,6 +240,29 @@ struct webclient_context
   FAR void *body_callback_arg;
   FAR const struct webclient_tls_ops *tls_ops;
   FAR void *tls_ctx;
+  unsigned int flags;
+
+  /* WEBCLIENT_FLAG_NON_BLOCKING tells webclient_perform() to
+   * use non-blocking I/O.
+   *
+   * If this flag is set, webclient_perform() returns -EAGAIN
+   * when it would otherwise block for network I/O. In that case,
+   * the application should either retry the operation later by calling
+   * webclient_perform() again, or abort it by calling webclient_abort().
+   * It can also use webclient_get_poll_info() to avoid busy-retrying.
+   *
+   * If this flag is set, it's the application's responsibility to
+   * implement a timeout.
+   *
+   * If the application specifies tls_ops, it's the application's
+   * responsibility to make the TLS implementation to use non-blocking I/O
+   * in addition to specifying this flag.
+   *
+   * Caveat: Even when this flag is set, the current implementation performs
+   * the name resolution in a blocking manner.
+   */
+
+#define	WEBCLIENT_FLAG_NON_BLOCKING	1U
```

Comment:
Placement of this preprocessor definition is not compliant to the coding standard.

Reply:
fixed